ground/src/main/java/com/google/android/ground/model/job/Job.kt

```diff
@@ -19,21 +19,28 @@ import android.graphics.Color
 import com.google.android.ground.model.task.Task
 import timber.log.Timber
 
-/**
- * @param suggestLoiTaskType the type of task used to suggest the LOI for this Job. Null if the job
- * is already associated with an LOI.
- */
 data class Job(
   val id: String,
   val style: Style? = null,
   val name: String? = null,
   val tasks: Map<String, Task> = mapOf(),
-  val suggestLoiTaskType: Task.Type? = null,
+  val strategy: DataCollectionStrategy = DataCollectionStrategy.MIXED
```

Comment:
Should this have a default? Don't we need this to be defined by survey organizers? Shouldn't it be a required argument, and shouldn't we error out if it's not available/provided?

Reply:
The default value was mainly for tests, but that's probably the wrong reason. Updating now.

Reply:
Doing this turned out to be more disruptive than I expected; basically we end up needing to defined it in all tests even where it's not relevant. Ideally we'd have a custom constructor with default values for tests, but since we don't have yet that we can apply UNKNOWN as a default based on your next suggestion